DopplerJobTest/DopplerCurrencyServiceTests.cs

```diff
@@ -210,18 +210,18 @@ public async Task DopplerCurrencyService__ShouldBeHttpStatusCodeOk_WhenDateIsHol
                     ItExpr.IsAny<CancellationToken>())
                 .ReturnsAsync(new HttpResponseMessage
                 {
-                    StatusCode = HttpStatusCode.BadRequest,
-                    Content = new StringContent("{\"success\":false,\"errors\":{\"No USD for this date\":[\"There are no pending USD currency for that date.\"]}}")
+                    StatusCode = HttpStatusCode.OK,
+                    Content = new StringContent(@"{'cotizationAvailable':false, 'date':'" + $"{DateTime.UtcNow:yyyy-MM-dd}" + "','currencyName':'Peso Argentino', 'currencyCode':'ARS'}")
                 });
 
+            var okResponse = 
```

Comment:
This variable is not used, you can remove it

Reply:
yes, sure